packages/devtools-source-map/src/source-map.js

```diff
@@ -195,6 +195,19 @@ async function getGeneratedLocation(
       column: location.column == null ? 0 : location.column,
       bias: SourceMapConsumer.LEAST_UPPER_BOUND
     });
+
+    if (map.originalPositionFor(match).line !== location.line) {
```

Comment:
@loganfsmyth Is this still relevant or should we bail?

Reply:
I think we should ditch this, since using the generated location + only showing things when there are more than a single column bp should hopefully avoid the need for it.